packages/breadcrumbs/src/styled/StyledBreadcrumbItem.spec.tsx

```diff
@@ -6,19 +6,41 @@
  */
 
 import React from 'react';
-import { render } from 'garden-test-utils';
+import { render, renderRtl } from 'garden-test-utils';
+import { getColor, DEFAULT_THEME } from '@zendeskgarden/react-theming';
 import { StyledBreadcrumbItem } from './StyledBreadcrumbItem';
 
 describe('StyledBreadcrumbItem', () => {
   it('renders default styling', () => {
     const { container } = render(<StyledBreadcrumbItem />);
 
-    expect(container.firstChild).toHaveClass('c-breadcrumb__item');
+    expect(container.firstChild).toHaveStyleRule('line-height', DEFAULT_THEME.lineHeights.md);
+    expect(container.firstChild).toHaveStyleRule('white-space', 'nowrap');
+    expect(container.firstChild).toHaveStyleRule('font-size', 'inherit');
   });
 
   it('renders current styling', () => {
     const { container } = render(<StyledBreadcrumbItem isCurrent />);
 
-    expect(container.firstChild).toHaveClass('is-current');
+    expect(container.firstChild).toHaveStyleRule(
+      'color',
+      getColor(DEFAULT_THEME.colors.neutralHue, 600)
+    );
+  });
+
+  it('renders the correct style for the ::after chevron pseudo element for LTR', () => {
+    const { container } = render(<StyledBreadcrumbItem />);
+
+    expect(container.firstChild).toHaveStyleRule('transform', 'none', {
+      modifier: '::after'
+    });
+  });
+
+  it('renders the correct style for the ::after chevron pseudo element for RTL', () => {
+    const { container } = renderRtl(<StyledBreadcrumbItem />);
+
+    expect(container.firstChild).toHaveStyleRule('transform', 'rotate(180deg)', {
+      modifier: '::after'
+    });
```

Comment:
The two tests above are newly added. They assert that users should see the correct chevron direction in the breadcrumb depending on their writing system (LTR vs RTL).